pkg/controller/egressgateway/egressgateway_controller.go

```diff
@@ -643,3 +699,11 @@ func getDegradedMsg(egw *operatorv1.EgressGateway) string {
 	}
 	return ""
 }
+
+func getEGWNamespacedNames(egws []operatorv1.EgressGateway) []string {
```

Comment:
This is a little confusing, since a `NamespacedName` is a common type used in the operator and this function doesn't return that type...